components/messaging/CurrentConversation.tsx

```diff
@@ -85,17 +85,17 @@ const CurrentConversation: React.FC = () => {
   }, [isScrolling, setIsScrolling]);
 
   return (
-    <div className='mb-10 flex h-screen flex-1 flex-col justify-end'>
+    <div className='conversation-height mb-10 flex flex-1 flex-col justify-between bg-[#fafaf9] shadow-inner'>
       <div
-        className='relative flex flex-col-reverse overflow-y-auto bg-stone-50'
```

Comment:
Watch out for this one ❗ We still need `flex-col-reverse` to keep messages at the bottom of CurrentConversation